docs/src/documentation/05-development/04-migration-guides/01-v11.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: v11
+description: How to migrate to Orbit 11.0.0
+redirect_from:
+  - /migration-guides/v11/
+---
+
+# Orbit Migration Guide v11
+
+This migration guide focuses on the process of migrating from Orbit v10.3.0 to v11.0, as some breaking changes were introduced.
+The best part is, migrating to v11 is incredibly simple!
+We've streamlined the process to ensure a smooth transition, making it easier than ever to benefit from the latest enhancements.
+
+## Breaking changes
+
+### Select has no longer `readOnly` property
+
+The `readOnly` prop was passed directly to the native HTML select element as an
```

Comment:
In some places we have "property" in others we have "prop". It would be nice to be consistent.